network_provisioning/examples/thread_prov/main/app_main.c

```diff
@@ -112,7 +112,7 @@ static esp_err_t example_get_sec2_verifier(const char **verifier, uint16_t *veri
 const int THREAD_ATTACHED_EVENT = BIT0;
 static EventGroupHandle_t thread_event_group;
 
-#define PROV_QR_VERSION         "v2"
+#define PROV_QR_VERSION         "v1"
```

Comment:
What is the requirement for this change?

Reply:
Would this be considered as a breaking change?

Reply:
Previously I want to use the QR code to distinguish the Thread device and Wi-Fi device, so I added a network_type parameter in QR code and increase the QR code version, but now we can use the capabilities to distinguish the Thread device and Wi-Fi device.

The QRcode of network provisioning or wifi provisioning will be the same. And the phone app can use the same step to establish the secure session with end devices. 
 
After that the phone app can read the provision version string to identify when the device is a network-prov device or a wifi-prov device. (network-prov device's version string will start with a prefix netprov)
 
For wifi-prov device, the phone app should use previous wifi-prov protobuf to commission the device to a Wi-Fi network.
 
For network-prov device, the phone app should read the capabilities to identify the device is a Wi-Fi device or a Thread device, and send the Wi-Fi/Thread credential with the new network-provision protobuf.